addons/azureSets/azuresets.lua

```diff
@@ -66,14 +67,36 @@ local bluPointsMax = nil
 local bluSlots = nil
 local get_blu_job_data = nil
 
+local spellsLookup = {}
+for spell in spells:it() do
+    spellsLookup[spell.english] = spell
+    spellsLookup[spell.english:lower()] = spell
+    spellsLookup[spell.japanese] = spell
+end
+
 function initialize()
     update_blu_info()
     update_current_spellset()
 end
 
-windower.register_event('load', initialize:cond(function() return windower.ffxi.get_info().logged_in end))
+windower.register_event('load', function()
+    local player = windower.ffxi.get_player()
+    local is_blu = player and player.main_job_id == BLU_JOB_ID
+    local logged_in = windower.ffxi.get_info()
+    if is_blu and logged_in then
+        initialize()
+    end
+end)
 
-windower.register_event('login', function() coroutine.schedule(initialize, 5) end)
+windower.register_event('incoming chunk', function(id, original, modified, injected, blocked) 
+    if id ~= 0x00A then
+        return
+    end
+    local packet = packets.parse('incoming', original)
+    if packet['Main Job'] == BLU_JOB_ID then
+       initialize()
+    end 
+end)
```

Comment:
I don't see why these were changed, but you could make it much easier by putting the sanity logic into the `initialize` function. Then you can reduce all events to this line:
```lua
windower.register_event('load', 'login', 'job_change', 'zone_change')
```

Though really, I don't see why you do this on zone change at all. If that `0x00A` thing was meant to be login, then use just the `login` event, not `incoming chunk` and filter for `0x00A`.